.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,68 @@
+name: Bug Report
+description: Report a bug encountered while using the Redpanda Helm Chart
+labels: kind/bug
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: What happened?
+      description: |
+        Please provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner.
+        If this matter is security related, please disclose it privately.
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: What did you expect to happen?
+    validations:
+      required: true
+
+  - type: textarea
+    id: repro
+    attributes:
+      label: How can we reproduce it (as minimally and precisely as possible)?. Please include values file.
+      value: |
+        <details>
+        
+        ```console
+        $ helm get values <redpanda-release-name> -n <redpanda-release-namespace> --all
+        # paste output here
+        ```
+        
+        </details>
+    validations:
+      required: true
+
+  - type: textarea
+    id: additional
+    attributes:
+      label: Anything else we need to know?
+
+  - type: textarea
+    id: chartVersion
+    attributes:
+      label: Chart version
+      value: |
+        <details>
+
+        ```console
+        $ helm -n <redpanda-release-namespace> list 
+        # paste output here
+        ```
```

Comment:
We have more than one chart. Probably need to know that, too.

Reply:
goood point.